libs/wingsdk/src/target-tf-gcp/app.ts

```diff
@@ -59,6 +70,8 @@ export class App extends CdktfApp {
       );
     }
 
+    this.zone = props.zone ?? process.env.GOOGLE_ZONE;
```

Comment:
Technically it is necessary to deploy `Table', but if it is not needed for other resources, so I do not raise an error here if it is undefined. Or maybe I should?

Reply:
Yes, we can - The zones are part of the region, so there is always at least one - 'a'.  That is a good suggestion, thank you - I committed it :)